django_downloadview/response.py

```diff
@@ -117,9 +117,10 @@ class DownloadResponse(StreamingHttpResponse):
       attributes (size, name, ...).
 
     """
+
     def __init__(self, file_instance, attachment=True, basename=None,
                  status=200, content_type=None, file_mimetype=None,
-                 file_encoding=None):
+                 file_encoding=None, extra_headers=None, **kwargs):
```

Comment:
Does ``**kwargs`` come from https://github.com/benoitbryon/django-downloadview/pull/109 ?
Is it really related to the use of ``args`` and ``kwargs`` in ``get_file()``?

About ``extra_headers``, I don't see the relationship with ``args`` and ``kwargs`` in ``get_file()``. If it is another feature, can you create another pull-request for this particular change?